dashboard/src/constants/browserIcons.ts

```diff
@@ -86,6 +86,26 @@ export const BROWSERS: Record<string, BrowserDef> = {
     localFile: 'ecosia.svg',
     match: [/ecosia/],
   },
+  google: {
+    label: 'Google',
+    localFile: 'google.svg',
+    match: [/google\s+search/],
+  },
+  twitter: {
+    label: 'Twitter',
+    localFile: 'twitter.svg',
+    match: [/twitter/],
+  },
+  facebook: {
+    label: 'Facebook',
+    localFile: 'facebook.svg',
+    match: [/\bfbios\b/],
+  },
+  appleMail: {
+    label: 'Apple Mail',
+    localFile: 'apple-mail.svg',
```

Comment:
The added SVG is called `mail.svg` and not `apple-mail.svg`, so I'd suggest renaming the file to `apple-mail.svg` 😄